templates/repo/settings/webhook/gitea.tmpl

```diff
@@ -35,6 +35,51 @@
 			<label for="secret">{{.locale.Tr "repo.settings.secret"}}</label>
 			<input id="secret" name="secret" type="password" value="{{.Webhook.Secret}}" autocomplete="off">
 		</div>
+
+		<div class="ui divider"></div>
+
+		<div class="auth-headers">
+			<h4>{{.locale.Tr "repo.settings.webhook.auth_header.section"}}</h4>
+			<div class="field">
+				<div class="ui checkbox">
+					<input class="hidden" name="auth_header_active" type="checkbox" tabindex="0" {{if .GiteaHook.AuthHeaderEnabled}}checked{{end}}>
```

Comment:
Does this `tabindex` have any purpose? There are input elements before this. I'd recommend not setting it and just let the natural browser-derived tabindex take over.

Reply:
😀 No, I assume that's just a copy/paste left-over.